routes/outbound-lims.xml

```diff
@@ -79,8 +111,8 @@
 
                 <!--create client(from ServiceRequest.subject) on SENAITE and save client UID property-->
 
-                <toD uri="{{senaite.baseUrl}}/@@API/senaite/v1/search?portal_type=Client&amp;getName=${exchangeProperty.patient-name}"/>
-
+                <toD uri="{{senaite.baseUrl}}/@@API/senaite/v1/search?portal_type=Client&amp;getName=${exchangeProperty.patient-name-unique}"/>
```

Comment:
Same thing here, we should avoid the `toD`as much as possible, and if must be, let's make sure a new endpoint don't get created for every patient.